seqr/views/apis/individual_api_tests.py

```diff
@@ -59,6 +60,18 @@
     'individualGuid': PM_REQUIRED_INDIVIDUAL_GUID, 'individualId': 'NA20889', 'familyId': '12', 'displayName': 'NA20889_a'
 }
 
+EXTERNAL_WORKSPACE_INDIVIDUAL_GUID = 'I000019_na21987'
+EXTERNAL_WORKSPACE_INDIVIDUAL_UPDATE_DATA = {
+    'individualGuid': EXTERNAL_WORKSPACE_INDIVIDUAL_GUID,
+    'individualId': 'NA21987',
+    'paternalGuid': 'I000018_na21234',
+    'maternalGuid': 'I000020_na65432',
```

Comment:
How about adding another test without parent `guids`?

Reply:
thats tested about 3 times already in the test, so no